### ISSUE TITLE
What happens if the phone goes to sleep?

### ISSUE THREAD

**Issue by hcientist** [TeehanLax/SwiftSocketIODemo#4]
If i have a socket.io connection from an iphone to a (node) server, and then the phone turns off its display, can the app still receive socketio messages? is anything special required to do this?

**Comment by brod-ie** [comment]
This isn't a library specific question but is in relation to the platform itself. [Read up here](https://developer.apple.com/library/ios/documentation/iPhone/Conceptual/iPhoneOSProgrammingGuide/BackgroundExecution/BackgroundExecution.html) on the background and suspended states

**Comment by Knights22** [comment]
No, only apps using voip have this option to run in the background. The best practice is to monitor when the app is going to the background and send an update to the server noting this. Then whenever that user gets a message default to push notifications.
